webpack.config.js

```diff
@@ -29,46 +29,33 @@ exec(`pybabel compile -d ${localePath} -D sphinx`);
  * Paths for various assets (sources and destinations)
  */
 
-const vendorVersions = {fontAwesome: require("@fortawesome/fontawesome-free/package.json").version};
-
 const scriptPath = resolve(__dirname, "src/pydata_sphinx_theme/assets/scripts");
-const staticPath = resolve(__dirname,"src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static");
+const staticPath = resolve(__dirname, "src/pydata_sphinx_theme/theme/pydata_sphinx_theme/static");
 const vendorPath = resolve(staticPath, "vendor");
-const faPath = {fontAwesome: resolve(vendorPath, "fontawesome", vendorVersions.fontAwesome)};
 
 /*******************************************************************************
  * functions to load the assets in the html head
  * the css, and js (preload/scripts) are digested for cache busting
  * the fonts are loaded from vendors
  */
 
-function stylesheet(css){ return `<link href="{{ pathto('_static/${css}', 1) }}?digest=${this.hash}" rel="stylesheet" />`;}
-function preload(js){ return `<link rel="preload" as="script" href="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}" />`;}
-function script(js){ return `<script src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`;}
-function font(woff2){ return `<link rel="preload" as="font" type="font/woff2" crossorigin href="{{ pathto('_static/${woff2}', 1) }}" />`;}
+function stylesheet(css) { return `<link href="{{ pathto('_static/${css}', 1) }}?digest=${this.hash}" rel="stylesheet" />`; }
+function preload(js) { return `<link rel="preload" as="script" href="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}" />`; }
+function script(js) { return `<script defer src="{{ pathto('_static/${js}', 1) }}?digest=${this.hash}"></script>`; }
 
 /*******************************************************************************
  * the assets to load in the macro
  */
- const theme_stylesheets = [
+const theme_stylesheets = [
   "styles/theme.css", // basic sphinx css
-  "styles/bootstrap.css", // all bootstrap 5 css with variable adjustments
   "styles/pydata-sphinx-theme.css", // all the css created for this specific theme
 ];
 const theme_scripts = [
   "scripts/bootstrap.js",
   "scripts/pydata-sphinx-theme.js",
 ];
-const fa_stylesheets = [
-  `vendor/fontawesome/${vendorVersions.fontAwesome}/css/all.min.css`,
-];
 const fa_scripts = [
-  `vendor/fontawesome/${vendorVersions.fontAwesome}/js/all.min.js`,
-];
-const fa_fonts = [
-  `vendor/fontawesome/${vendorVersions.fontAwesome}/webfonts/fa-solid-900.woff2`,
-  `vendor/fontawesome/${vendorVersions.fontAwesome}/webfonts/fa-brands-400.woff2`,
-  `vendor/fontawesome/${vendorVersions.fontAwesome}/webfonts/fa-regular-400.woff2`,
+  "scripts/fontawesome.js",
```

Comment:
don't you ignore it in `.gitignore` now ? Should we ? This forces contributor to build.. which I'm ok with, but maybe keep it ? Or maybe i'm mistaken and this one is not th one in `.gitgnore`

Reply:
A contributor will always need to build if they are working on the theme anyway, so I followed the pattern we have for other assets.